src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -186,7 +184,7 @@
                                                     <div class="form-group form-inline">
                                                         <div class="form-group text-muted">
                                                             <p>
-                                                                Comment about <%= InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name) %>:
+                                                                Comment about <%= InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name + " (" + data.student.team + ", " + data.student.email + ")") %>:
                                                             </p>
```

Comment:
I see. That is after clicking the edit button. Before clicking 'edit' it is like this:
<img width="1046" alt="screen shot 2015-07-16 at 10 50 35 am" src="https://cloud.githubusercontent.com/assets/7547452/8714522/a659c160-2ba8-11e5-8957-e76adc40e5bd.png">
In both cases, the email is showed once.

I guess we have to choose between consistency and duplicate display in this case.